src/components/DayTabsList.vue

```diff
@@ -101,7 +102,7 @@ export default {
         { day: "Today", id: -1 },
         ...filteredTabs.slice(index),
         ...filteredTabs.slice(0, index),
-      ];
+      ].slice(0, -1);
```

Comment:
So i've been thinking about this computed property and also looking at the designs and I *think* the tabs are supposed to work a bit differently 🤔 

On the design we always have `Today` as a first tab, and then we have a few buttons for the *next days* after today:
* If today would be Monday then those would be Tue, Wed, Thu, Fri, Sat
* If today is Friday would be Friday then those would be Sat, Sun, Mon, Tue, Wed

Soooo I think we need to generate a list of tabs for upcoming days (including today's weekday) - this would also be much better UX wise - users can easily think about about current weekday and upcoming weekdays 💡 

What do you think?

Reply:
I'm not sure if I understand you. Please check the video and tell me if it should work differently. 🤔
https://user-images.githubusercontent.com/80581523/164689693-d68753a4-c113-45a3-bbe8-e40a01e18cae.mov

Reply:
Ohhh okay I now see that it works - my bad! 🙈 
Taken a second look at the code and now I see it's absolutely working properly - It's quite tricky but clever! 😄
If anything I'd just add a small comment that this essentially gives us today + 5 next days as tabs for clarity 👍